src/stim/util_top/reference_sample_tree.perf.cc

```diff
@@ -0,0 +1,23 @@
+#include "stim/util_top/reference_sample_tree.h"
+
+#include "stim/gen/circuit_gen_params.h"
+#include "stim/gen/gen_surface_code.h"
+#include "stim/perf.perf.h"
+
+using namespace stim;
+
+BENCHMARK(reference_sample_tree_surface_code_d31_r1000000000) {
```

Comment:
You may also want to try and benchmark against circuits with nested loops.

Reply:
Done.